template/package.json

```diff
@@ -21,23 +21,23 @@
     "@babel/plugin-transform-runtime": "^7.16.4",
     "@babel/preset-env": "^7.16.4",
     "@studiometa/browserslist-config": "^1.0.1",
-    "@studiometa/eslint-config": "^2.1.3",
-    "@studiometa/prettier-config": "^2.0.1",
+    "@studiometa/eslint-config": "^3.0.1",
+    "@studiometa/prettier-config": "^2.1.1",
     "@studiometa/prettier-formatter-gitlab": "^1.1.1",
     "@studiometa/stylelint-config": "^2.0.0",
-    "@studiometa/tailwind-config": "^1.0.4",
-    "@studiometa/webpack-config": "^3.0.2",
+    "@studiometa/tailwind-config": "^1.1.0",
+    "@studiometa/webpack-config": "^3.1.2",
     "eslint": "^7.32.0",
     "eslint-formatter-gitlab": "^3.0.0",
     "postcss": "^8.3.11",
     "prettier": "^2.4.1",
     "prettier-plugin-twig-melody": "^0.4.6",
     "stylelint": "^13.13.1",
     "stylelint-formatter-gitlab": "^1.0.1",
-    "tailwindcss": "^2.1.19"
+    "tailwindcss": "^3.0.23"
   },
   "dependencies": {
-    "@studiometa/js-toolkit": "^2.0.0-beta.0",
-    "@studiometa/ui": "^0.1.0"
+    "@studiometa/js-toolkit": "^2.0.1",
```

Comment:
Can we update to the latest version: 2.2.2 ? This should not have any impact right @titouanmathis ?